doc/reference/storage_drivers.md

```diff
@@ -82,6 +82,7 @@ Optimized backup (import/export)            | ❌        | ✅   | ❌     | ✅
 Copy on write                               | ❌        | ✅   | ✅     | ✅     | ✅       | ✅     | ➖          | ✅              | ✅          | ✅
 Block based                                 | ❌        | ❌   | ✅     | ❌      | ✅      | ❌     | ➖          | ✅              | ✅          | ✅
 Instant cloning                             | ❌        | ✅   | ✅     | ✅     | ✅       | ✅     | ➖          | ❌              | ✅          | ❌
+Idmapped mounts                             | ❌        | ✅   | ❌     | ✅     | ❌       | ✅     | ➖          | ❌              | ❌          | ❌
```

Comment:
What is your definition of idmapped mount support here? Because I am seeing several missing ones, such as dir, lvm, ceph, dell, pure, and hpe.

Reply:
I'll have a chat about it with Aleks as my understanding is probably very off then.